rllib/algorithms/algorithm.py

```diff
@@ -730,15 +741,6 @@ def evaluate(
                 episodes left to run. It's used to find out whether
                 evaluation should continue.
         """
-        # In case we are evaluating (in a thread) parallel to training,
-        # we may have to re-enable eager mode here (gets disabled in the
-        # thread).
-        if (
-            self.config.get("framework") in ["tf2", "tfe"]
-            and not tf.executing_eagerly()
-        ):
-            tf1.enable_eager_execution()
```

Comment:
Resolved?

Reply:
Ah, thanks. Yeah, it's really not needed here. `evaluate` is never called from within a thread (yes, it could be run on a different node/process b/c it's on some remote eval worker, but never inside a thread).